src/constants/editorconfigMappings.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+	charset: {
+		name: 'encoding',
+		types: ['string'],
+		regexp: /^.*$/,
```

Comment:
Allow only values that node can parse. Later we can implement the fallback discussed in #48 
```
regexp: /^(?:latin1|utf-8|utf-16le)$/,
```

Reply:
I am going to merge this right now. But we can keep this suggestion for the required changes in #48. 👍